static/scripts/dashboard/dashboardCtrl.js

```diff
@@ -2444,7 +2448,7 @@
 
       var bag = {
         name: 'sshKeys',
-        masterName: $scope.vm.installForm.sshKeys['ssh-key'].masterName,
+        masterName: $scope.vm.installForm.sshKeys['sshKey'].masterName,
```

Comment:
I'm not sure about how this works. We're not creating keys if `ssh-key` is already present. So, those keys will not show up anymore right?